tools/utility/dockerise/run.py

```diff
@@ -91,7 +91,7 @@ def _setup_volume(volume_name, clean_volume):
     return volume
 
 
-def _setup_internal_image(image, rebuild, clean_volume):
+def _setup_internal_image(image, rebuild, clean_volume, clean_uv_cache=False):
```

Comment:
Should be able to handle clean_uv_cache the same as clean_volume so it shouldn't need a default